app/grandchallenge/reader_studies/forms.py

```diff
@@ -310,6 +337,52 @@ def interface_choices(self):
     def initial_interface(self):
         return self.interface_choices().first()
 
+    def widget_choices(self):
+        answer_type = self["answer_type"].value()
+        choices = [("", "Default")]
```

Comment:
You could use `BLANK_CHOICE_DASH` for consistency here.

Reply:
When Chris and I discussed this, we thought that `Default` would be clearer for the user than a dashed line. The dashed line indicates that no widget is used, which is not really the case, right?

Reply:
It means that the user has not made a choice, this pattern exists in many places and we should be consistent. Other places have tried to re-invent this, it is a mess.